Orange/widgets/data/owpythonscript.py

```diff
@@ -386,7 +386,7 @@ def __init__(self):
         self.in_learner = None
         self.in_classifier = None
         self.in_object = None
-        self.auto_execute = False
```

Comment:
This is a bad idea. If it is supposed to be a setting, changing it to false/true in __init__ overrides the saved value.